spec/acceptance/suites/rpm_el7/01_simp_server_spec.rb

```diff
@@ -89,16 +47,16 @@ def internet_yumrepos(host, reponame)
     masters.each do |master|
       it 'should set up SIMP repositories' do
         master.install_package('epel-release')
-
-        tarball = find_tarball
-        if tarball.nil? or tarball.empty?
-          internet_yumrepos(master, find_reponame)
-        else
-          tarball_yumrepos(master, tarball)
-        end
+        setup_repo(master)
         on(master, 'yum makecache')
       end
 
+      use_puppet_repo = ENV['BEAKER_puppet_repo'] || false
+
+      if use_puppet_repo
+        master.install_package('http://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm')
```

Comment:
Do you mean the ones in the dependancy libraries?  Because it is unless you set the BEAKER_puppet_repo = true, but if you don't set it it should not use the puppet repo.